### DISCUSSION TITLE
Make it less verbose

### DISCUSSION THREAD

**Discussion by pkese** [microsoft/vscode-makefile-tools#566] [General]
I have no clue what this is, and what it wants from me.
![image](https://github.com/microsoft/vscode-makefile-tools/assets/278492/bf064cf5-8699-4a5e-9681-6e2e84f91e54)

or this
![image](https://github.com/microsoft/vscode-makefile-tools/assets/278492/dca705ca-f0b8-4604-aed5-3e34cf791b85)

These 'Makefile.tools' extension is making interacting with vscode miserable ever since I installed it.

Not everyone is using Makefiles for C/C++ projects.

And if you can't accept that people are using Makefiles for other things than C/C++, then please at least don't force people to install your plugin each time a Makefile appears in a repository. 
![image](https://github.com/microsoft/vscode-makefile-tools/assets/278492/4a46c4ab-40d1-4cef-8aec-abeff37d0aa6)

**Comment by sinemakinci1** [comment]
Just wanted to update the thread here that we are redesigning the initial configure experience, and proposing removing the dry-run popup for ease of use for users and so popups do not persist. 

If you would like to add any additional comments on this effort, please see our issue #572

**Comment by gcampbell-msft** [comment]
This is modified / resolve so I'll close this discussion.
